packages/fast-sketch-library/readme.md

```diff
@@ -0,0 +1,40 @@
+# FAST-DNA sketch library
```

Comment:
Remove "-DNA" to be consistent with how we are naming other components as a title of the readme. But, very happy to see a readme and the right details to use.  :-)

Reply:
done